Jenkinsfile.rh.release

```diff
@@ -24,9 +24,51 @@ properties([
   ]),
 ])
 
+String version = ''
+String checksum = ''
+
+void updateIQServerVersionAndChecksum(String version, String checksum) {
+  def dockerFile = readFile(file: 'Dockerfile.rh')
+  def metaShortVersionRegex = /(release=")(\d\.\d{1,3}\.\d)(" \\)/
+  def versionRegex = /(ARG IQ_SERVER_VERSION=)(\d\.\d{1,3}\.\d\-\d{2})/
```

Comment:
Not sure why this was ever done like this, but it's a complete anti-pattern. The entire point of build-args is to override them on the Docker build invocation. This would be much simpler if you just pass these in to your script invoking Docker and leave this file alone. 
https://docs.docker.com/engine/reference/commandline/build/#build-arg

Reply:
Commit: a2a41fa10702ac5eaf527a8818f39a19baf2e028
Does this make more sense for you?
Would this replace the values for `IQ_SERVER_VERSION` and `IQ_SERVER_SHA256` wherever they appear in `Dockerfile.rh` ?

Reply:
+1 to Kelly's comment. It was inherited in other images to minimize the number of changes but in hindisght we should have gone this route which is what MTIQ and IQ (I think) do. We are also doing this on the NVS side of the house.